### ISSUE TITLE
seasonal thermometor icons

### ISSUE THREAD

**Issue by greenring** [Templarian/MaterialDesign#6591]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Icons for a thermostat, where you can select different temperatures for different seasons.

### Give us an example of what this icon might look like.

mdi:sun-thermometer is perfect for summer.

I would expect something like 

- mdi:snowflake-thermometer for winter, which would match mdi:snowflake
- mdi:flower-thermometer for spring, which would match mdi:flower
- mdi:leaf-thermometer for autumn, which would match mdi:leaf

HomeAssistant uses the following in their seasonal sensor:
    STATE_SPRING: "mdi:flower",
    STATE_SUMMER: "mdi:sunglasses",
    STATE_AUTUMN: "mdi:leaf",
    STATE_WINTER: "mdi:snowflake",

Because of this, you might also consider mdi:sunglasses-thermometer as well.

**Comment by greenring** [comment]
Forgot to link to HA src: https://github.com/home-assistant/core/blob/dev/homeassistant/components/season/sensor.py

**Comment by goyney** [comment]
I don't think this really works completely. `sun-thermometer` and `snowflake-thermometer` seem pretty useful, and we should probably add the later (for freeze point), but "the temperate of a flower" and the "temperature of a leaf" really don't make sense.

Also, while it's easy to cut out part of the sun/snowflake (and we've done it in the past), it's not with the flower or the leaf without damaging the readability of the base icon.

**Comment by brendankmurphy** [comment]
Hi @mririgoyen -

One consideration, at least for snowflake-thermometer, might be to match the thermometer style to the temperature consistent with the snowflake icon.

As it stands, the snowflake on the snowflake-thermometer icon is paired with the standard thermometer, which appears to be reading somewhat high.  Alternatively, there is "thermometer-low" icon which would likely be better paired with the snowflake icon for the snowflake-thermometer icon.

Thanks!
